rails_app/.rubocop.yml

```diff
@@ -0,0 +1,66 @@
+# The guiding principle of classes is SRP, SRP can't be accurately measured by LoC
+Metrics/ClassLength:
+  Max: 1500
```

Comment:
Damn boy, thats a lot of lines for a single class

Reply:
thought its self explanatory comment above